src/timeline.ts

```diff
@@ -0,0 +1,114 @@
+/**
+ * Created by Martin on 04.01.2018.
+ */
+
+import {IMalevoDataset, IMalevoDatasetCollection, IMalevoEpochInfo} from './malevo_dataset';
+import * as d3 from 'd3';
+import * as events from 'phovea_core/src/event';
+import {AppConstants} from './app_constants';
+import * as ajax from 'phovea_core/src/ajax';
+import {INumericalMatrix} from 'phovea_core/src/matrix';
+import {IDragSelection} from './range_selector';
+import RangeSelector from './range_selector';
```

Comment:
Do not use the default export! 
Write: `import {TimelineRangeSelector} from './range_selector';`